api/cpp/cbindgen.rs

```diff
@@ -604,11 +605,18 @@ fn gen_corelib(
     friend inline LayoutInfo operator+(const LayoutInfo &a, const LayoutInfo &b) { return a.merge(b); }
     friend bool operator==(const LayoutInfo&, const LayoutInfo&) = default;".into(),
     );
+    config.export.body.insert(
+        "WindowEvent".to_owned(),
+        "/* Some members of the WindowEvent enum have destructors (with SharedString), but thankfully we don't use these so we can have an empty constructor */
```

Comment:
Ah yeah, this has caused memory leaks before (with the imageinner IIRC). Tricky, but I agree with your use of the enum directly instead of the wrapper functions.